kloppy/domain/models/event.py

```diff
@@ -126,6 +126,42 @@ class SetPieceQualifier(EnumQualifier):
     value: SetPieceType
 
 
```

Comment:
I can imagine some of these might change over time to a more generic naming. Some sound a bit cryptic (like launch and smart pass). Have you seen similar classifiers in other formats @bdagnino

Reply:
Those are the ones coming from WyScout. We can find better names. Maybe you have a suggestion @koenvo or @bdagnino ?